### PR TITLE
Robotx merge branch

### PR DESCRIPTION
**Pull request by alexperez33**

### CODE REVIEW

File: NaviGator/mission_control/navigator_launch/launch/perception/classifier.launch [hunk 1]
```diff
@@ -3,18 +3,22 @@
...
     <arg name="out_topic_model1" default="detections_model1" />
     <arg name="out_topic_model2" default="detections_model2" />
-    <arg name="conf_thresh" default="0.75" />
+    <arg name="out_topic_model3" default="stc_detections_model" />
+    <arg name="conf_thresh" default="0.25" />
     <arg name="image_size" default="640" />
-    <arg name="device" default="cuda" />
-    <arg name="use_yolo_model1" default="True"/>
-    <arg name="use_yolo_model2" default="True"/>
+    <arg name="device" default="cpu" />
```
Comment (cbrxyz) [inline NaviGator/mission_control/navigator_launch/launch/perception/classifier.launch:18]: Should the device be set to `cuda` over `cpu` for better performance?

File: NaviGator/mission_control/navigator_launch/launch/vrx/vrx_classifier.launch [hunk 2]
```diff
@@ -6,8 +6,8 @@
     <arg name="out_topic" default="detections" />
     <arg name="conf_thresh" default="0.75" />
     <arg name="image_size" default="640" />
-    <arg name="device" default="cuda" />
-    <arg name="use_yolo" default="True"/>
+    <arg name="device" default="cpu" />
```
Comment (cbrxyz) [inline NaviGator/mission_control/navigator_launch/launch/vrx/vrx_classifier.launch:9]: Same

File: mil_common/utils/mil_tools/mil_ros_tools/extract_bag_images.py [hunk 3]
```diff
@@ -197,7 +197,7 @@ def _save_img(self, msg, time, image_dir, prefix=""):
         if self.encoding == ImageProc.COLOR or self.encoding == ImageProc.RECT_COLOR:
             img = cvtColor2(img, self.image_set.color_encoding, "bgr8")
 
-        img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
+        # img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
```
Comment (cbrxyz) [inline mil_common/utils/mil_tools/mil_ros_tools/extract_bag_images.py:200]: Weird to leave this as commented out, we should add either add an explanation comment or just remove this line.

File: NaviGator/mission_control/navigator_missions/navigator_missions/scan_the_code.py [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+from mil_tools import numpy_to_pointcloud2 as np2pc2
+from mil_tools import rosmsg_to_numpy
+from mil_vision_tools.cv_tools import contour_mask, rect_from_roi, roi_enclosing_points
+from navigator_msgs.msg import ScanTheCode
+from sensor_msgs.msg import Image, PointCloud2
+from std_srvs.srv import SetBoolRequest
+
+from .navigator import Navigator
+
+LED_PANEL_MAX = 0.1
```
Comment (cbrxyz) [inline NaviGator/mission_control/navigator_missions/navigator_missions/scan_the_code.py:20]: Units? We should add this as a comment following the number like
```python
CONSTANT_WITH_UNITS = 0.6 # meters
```

**Review by cbrxyz — CHANGES_REQUESTED** [review]
Minor things

File: NaviGator/mission_control/navigator_missions/navigator_missions/scan_the_code.py (hunk 4 above)
Comment (alexperez33) [reply NaviGator/mission_control/navigator_missions/navigator_missions/scan_the_code.py:20]: agreed

File: NaviGator/mission_control/navigator_launch/launch/perception/classifier.launch (hunk 1 above)
Comment (alexperez33) [reply NaviGator/mission_control/navigator_launch/launch/perception/classifier.launch:18]: we agree on cpu

File: NaviGator/mission_control/navigator_launch/launch/vrx/vrx_classifier.launch (hunk 2 above)
Comment (alexperez33) [reply NaviGator/mission_control/navigator_launch/launch/vrx/vrx_classifier.launch:9]: same as above

**Review by cbrxyz — APPROVED** [review]

**Pull request merged**